setup.py

```diff
@@ -61,7 +61,7 @@
     cmdclass= cmdclass,
     packages=setuptools.find_packages(),
     install_requires=[
-        "jupyterlab>=3.0.0rc5,==3.*",
+        "jupyterlab>=3.0.0rc5,==3.*", "jupyterlab_widgets>=1.0.0a6"
```

Comment:
For now it's probably simpler to require `jupyterlab_widgets` instead of trying to handle `@jupyter-widgets/jupyterlab-manager` as an optional shared package